TROUBLESHOOTING.md

```diff
@@ -0,0 +1,81 @@
+# Troubleshooting guide for the Cluster-Insight server
+
+## The server does not respond
+1.  Verify that the server is running.
+    1. Login to the master node.
+    0. Try to access the Cluster-Insight server from the local machine with
+      the command: `curl http://localhost:5555`.
+      If you get the HTML contents of the Cluster-Insight's home page,
+      the the server is functional. Follow the instructions below for other
+      causes for the problem.
+    2. List the running containers with the command:
+       `sudo docker ps`.
+    3. If the `sudo docker ps` command failed, you will need to restart Docker
+       with the command: `sudo service docker start`.
+    4. If the container `kubernetes/cluster-insight` is not listed in the
+       output of `sudo docker ps`, you will have to start it.
+       Please follow the instructions in the [README](README.md) file.
+       You may need to
+       delete a previous dead container with the same name if the `docker run`
+       command fails.
+    5. If the container `kubernetes/cluster-insight` is listed as running,
+       then it is probably stuck.
+       Read its log by the command:
+       `sudo docker log CONTAINER_ID`. Pay attention to the messages at the
```

Comment:
Should be `sudo docker logs CONTAINER_ID` (`logs -f` is better in order to see the results of the log on hitting the API)